ci-operator/config/openshift/elasticsearch-operator/master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+images:
+- from: base
+  to: origin-elasticsearch-logging-operator
+resources:
+  '*':
+    limits:
+      cpu: '2'
+      memory: 4Gi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
+  tag_overrides: {}
+test_base_image:
```

Comment:
`test_base_image` is deprecated, use `build_root.image_stream_tag` instead
https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#build_root